app/models/scanned_book.rb

```diff
@@ -5,6 +5,7 @@ class ScannedBook < ActiveFedora::Base
   include ::CurationConcerns::BasicMetadata
   include ::NoidBehaviors
 
+  property :sort_title, predicate: ::RDF::URI.new("http://library.princeton.edu/terms/primary_title"), multiple: false
```

Comment:
I feel like I've looked for one before without luck or range/domain restrictions, but it's been a while.

It's one of those things that's far too practical for linked data :wink: